src/App/App.js

```diff
@@ -137,6 +116,27 @@ class App extends Component {
   return changedData;
  }
 
+ login = async (loginState) => {
+  const response = await API.postData(loginState);
+  const user = await response.json();
+  if (response.status === 201) {
```

Comment:
This kind of error handling, where you're checking the status of the request and throwing if it's not a 2xx level, should be happening on your API class. You could even make a method that checks the `response.ok` property and call it in each of your other methods.